src/features/WeatherMood/components/Weather/WeatherRecommend.module.scss

```
@@ -130,6 +131,39 @@
   }
 }
 
+.weather-skeleton {
+  &__icon {
+    @extend %skeleton;
+    width: 72px;
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)